README.md

```diff
@@ -13,3 +14,14 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+The three reasons why asymptotic analysis may be misleading is:
+1: The amount of constants, these are disregarded factors. With a high asymptotic complexity an algorithm might have a lot of constant factors so they're not ideal for small input sizes becasue they're just approximations.
+2: Physical resstraints like hardware aren't taken into account becasue the analysis is abstract and may preform better or worse on different machines.
```

Comment:
Would that just amount to a constant, making it the same reason as the one above?